Loading LIBD module for git-status-size/github
Loading LIBD module for git-lfs/2.8.0
Loading LIBD module for rmate/1.5.9
Loading LIBD module for ruby/2.6.4p104
Loading conda_R/devel
**** Job starts ****
Mon Jan  2 07:47:33 EST 2023
**** JHPCE info ****
User: sparthib
Job id: 1382756
Job name: magma_heatmap
Hostname: compute-095.cm.cluster
Task id: 3
Unloading conda_R/devel
Loading conda_R/4.2

The following have been reloaded with a version change:
  1) conda/3-4.6.14 => conda/3-4.11.0     2) conda_R/devel => conda_R/4.2


Currently Loaded Modules:
  1) matlab/R2019a     6) COMMUNITY_CENTOS7_DEFAULT_ENV  11) ruby/2.6.4p104
  2) stata/17          7) JHPCE_CENTOS7_DEFAULT_ENV      12) rmate/1.5.9
  3) JHPCE_tools/1.0   8) git-status-size/github         13) conda/3-4.11.0
  4) sge/8.1.9         9) git/2.28.0                     14) conda_R/4.2
  5) gcc/4.4.7        10) git-lfs/2.8.0

 

here() starts at /dcs04/lieber/lcolladotor/with10x_LIBD001/Visium_IF_AD

Attaching package: ‘dplyr’

The following objects are masked from ‘package:stats’:

    filter, lag

The following objects are masked from ‘package:base’:

    intersect, setdiff, setequal, union

[1] "/dcs04/lieber/lcolladotor/with10x_LIBD001/Visium_IF_AD"
[1] 3
[1] "200"
Using Region, Pathology_Type as id variables
[1] 21  4
Using Region, Pathology_Type as id variables

TRUE 
  21 
  Region Pathology_Type           GWAS      Beta       P P.adj.fdr
1    ITC             Ab AD.Jansen.2019  0.054228 0.16333 0.4199236
2    ITC           both AD.Jansen.2019 -0.066124 0.87265 0.8775400
3    ITC        next_Ab AD.Jansen.2019 -0.067298 0.87754 0.8775400
4    ITC      next_both AD.Jansen.2019  0.050202 0.18025 0.4199236
5    ITC        next_pT AD.Jansen.2019 -0.034586 0.72660 0.8030842
6    ITC           none AD.Jansen.2019  0.043263 0.21561 0.4199236
Warning message:
In dir.create(here("plots", "magma")) :
  '/dcs04/lieber/lcolladotor/with10x_LIBD001/Visium_IF_AD/plots/magma' already exists
Error in image.plot(x = seq(0, ncol(wide_p), by = 1), y = clusterHeights,  : 
  could not find function "image.plot"
Calls: MAGMAplot
Execution halted
**** Job ends ****
Mon Jan  2 07:47:38 EST 2023
Loading LIBD module for git-status-size/github
Loading LIBD module for git-lfs/2.8.0
Loading LIBD module for rmate/1.5.9
Loading LIBD module for ruby/2.6.4p104
Loading conda_R/devel
**** Job starts ****
Mon Jan  2 08:02:35 EST 2023
**** JHPCE info ****
User: sparthib
Job id: 1382757
Job name: magma_heatmap
Hostname: compute-095.cm.cluster
Task id: 3
Unloading conda_R/devel
Loading conda_R/4.2

The following have been reloaded with a version change:
  1) conda/3-4.6.14 => conda/3-4.11.0     2) conda_R/devel => conda_R/4.2


Currently Loaded Modules:
  1) matlab/R2019a     6) COMMUNITY_CENTOS7_DEFAULT_ENV  11) ruby/2.6.4p104
  2) stata/17          7) JHPCE_CENTOS7_DEFAULT_ENV      12) rmate/1.5.9
  3) JHPCE_tools/1.0   8) git-status-size/github         13) conda/3-4.11.0
  4) sge/8.1.9         9) git/2.28.0                     14) conda_R/4.2
  5) gcc/4.4.7        10) git-lfs/2.8.0

 

here() starts at /dcs04/lieber/lcolladotor/with10x_LIBD001/Visium_IF_AD

Attaching package: ‘dplyr’

The following objects are masked from ‘package:stats’:

    filter, lag

The following objects are masked from ‘package:base’:

    intersect, setdiff, setequal, union

[1] "/dcs04/lieber/lcolladotor/with10x_LIBD001/Visium_IF_AD"
[1] 3
[1] "200"
Using Region, Pathology_Type as id variables
[1] 21  4
Using Region, Pathology_Type as id variables

TRUE 
  21 
  Region Pathology_Type           GWAS      Beta       P P.adj.fdr
1    ITC             Ab AD.Jansen.2019  0.054228 0.16333 0.4199236
2    ITC           both AD.Jansen.2019 -0.066124 0.87265 0.8775400
3    ITC        next_Ab AD.Jansen.2019 -0.067298 0.87754 0.8775400
4    ITC      next_both AD.Jansen.2019  0.050202 0.18025 0.4199236
5    ITC        next_pT AD.Jansen.2019 -0.034586 0.72660 0.8030842
6    ITC           none AD.Jansen.2019  0.043263 0.21561 0.4199236
Warning message:
In dir.create(here("plots", "magma")) :
  '/dcs04/lieber/lcolladotor/with10x_LIBD001/Visium_IF_AD/plots/magma' already exists
Error in image.plot(x = seq(0, ncol(wide_p), by = 1), y = clusterHeights,  : 
  could not find function "image.plot"
Calls: MAGMAplot
Execution halted
**** Job ends ****
Mon Jan  2 08:02:49 EST 2023
Loading LIBD module for git-status-size/github
Loading LIBD module for git-lfs/2.8.0
Loading LIBD module for rmate/1.5.9
Loading LIBD module for ruby/2.6.4p104
Loading conda_R/devel
**** Job starts ****
Mon Jan  2 08:09:34 EST 2023
**** JHPCE info ****
User: sparthib
Job id: 1382758
Job name: magma_heatmap
Hostname: compute-095.cm.cluster
Task id: 3
Unloading conda_R/devel
Loading conda_R/4.2

The following have been reloaded with a version change:
  1) conda/3-4.6.14 => conda/3-4.11.0     2) conda_R/devel => conda_R/4.2


Currently Loaded Modules:
  1) matlab/R2019a     6) COMMUNITY_CENTOS7_DEFAULT_ENV  11) ruby/2.6.4p104
  2) stata/17          7) JHPCE_CENTOS7_DEFAULT_ENV      12) rmate/1.5.9
  3) JHPCE_tools/1.0   8) git-status-size/github         13) conda/3-4.11.0
  4) sge/8.1.9         9) git/2.28.0                     14) conda_R/4.2
  5) gcc/4.4.7        10) git-lfs/2.8.0

 

here() starts at /dcs04/lieber/lcolladotor/with10x_LIBD001/Visium_IF_AD

Attaching package: ‘dplyr’

The following objects are masked from ‘package:stats’:

    filter, lag

The following objects are masked from ‘package:base’:

    intersect, setdiff, setequal, union

[1] "/dcs04/lieber/lcolladotor/with10x_LIBD001/Visium_IF_AD"
[1] 3
[1] "200"
Using Region, Pathology_Type as id variables
[1] 21  4
Using Region, Pathology_Type as id variables

TRUE 
  21 
  Region Pathology_Type           GWAS      Beta       P P.adj.fdr
1    ITC             Ab AD.Jansen.2019  0.054228 0.16333 0.4199236
2    ITC           both AD.Jansen.2019 -0.066124 0.87265 0.8775400
3    ITC        next_Ab AD.Jansen.2019 -0.067298 0.87754 0.8775400
4    ITC      next_both AD.Jansen.2019  0.050202 0.18025 0.4199236
5    ITC        next_pT AD.Jansen.2019 -0.034586 0.72660 0.8030842
6    ITC           none AD.Jansen.2019  0.043263 0.21561 0.4199236
Warning message:
In dir.create(here("plots", "magma")) :
  '/dcs04/lieber/lcolladotor/with10x_LIBD001/Visium_IF_AD/plots/magma' already exists
Error in image.plot(x = seq(0, ncol(wide_p), by = 1), y = clusterHeights,  : 
  could not find function "image.plot"
Calls: MAGMAplot
Execution halted
**** Job ends ****
Mon Jan  2 08:09:43 EST 2023
